common/djangoapps/student/forms.py

```diff
@@ -22,4 +28,44 @@ def clean_email(self):
         if any((user.password == UNUSABLE_PASSWORD)
                for user in self.users_cache):
             raise forms.ValidationError(self.error_messages['unusable'])
-        return email
\ No newline at end of file
+        return email
+
+    # pylint: disable=W0221
```

Comment:
Please disable using the message, not the error code (for example, `pylint: disable=broad-except`). That makes it clearer as to what we're actually disabling.